crypto/src/hash.rs

```diff
@@ -540,6 +540,13 @@ macro_rules! pk_with_hash {
                 Ok(typed_hash)
             }
         }
+
+        impl From<$pk> for $pkh {
+            fn from(source: $pk) -> Self {
+                let hash = blake2b::digest_160(&source.0);
+                Self::from_bytes(&hash).unwrap()
```

Comment:
```
source.pk_hash().unwrap()
```

Reply:
Done. I have amended the commit, see 33b9179e30b26027236cd825ba97386568160c60.